ui/app/AppLayouts/Wallet/popups/simpleSend/SimpleSendModal.qml

```diff
@@ -1,17 +1,495 @@
+    Expected model structure:
+    - name: name of account
+    - address: wallet address
+    - color: color of the account
+    - emoji: emoji selected for the account
+    - currencyBalance: total currency balance in CurrencyAmount
+    - accountBalance: balance of selected token + selected chain
+    **/
+    required property var accountsModel
+    /**
+    Expected model structure:
+    - tokensKey: unique string ID of the token (asset); e.g. "ETH" or contract address
+    - name: user visible token name (e.g. "Ethereum")
+    - symbol: user visible token symbol (e.g. "ETH")
+    - decimals: number of decimal places
+    - communityId:optional; ID of the community this token belongs to, if any
+    - marketDetails: object containing props like `currencyPrice` for the computed values below
+    - balances: submodel[ chainId:int, account:string, balance:BigIntString, iconUrl:string ]
+    - currentBalance: amount of tokens
+    - currencyBalance: e.g. `1000.42` in user's fiat currency
+    - currencyBalanceAsString: e.g. "1 000,42 CZK" formatted as a string according to the user's locale
+    - balanceAsString: `1.42` formatted as e.g. "1,42" in user's locale
+    - iconSource: string
+    **/
+    required property var assetsModel
+    /**
+    Expected model structure:
+    - groupName: group name (from collection or community name)
+    - icon: from imageUrl or mediaUrl
+    - type: can be "community" or "other"
+    - subitems: submodel of collectibles/collections of the group
+    - key: key of collection (community type) or collectible (other type)
+    - name: name of the subitem (of collectible or collection)
+    - balance: balance of collection (in case of community collectibles)
+               or collectible (in case of ERC-1155)
+    - icon: icon of the subitem
+    **/
+    required property var collectiblesModel
+    /**
+    Expected model structure:
+    - chainId: network chain id
+    - chainName: name of network
+    - iconUrl: network icon url
+    Only networks valid as per mainnet/testnet selection
+    **/
+    required property var networksModel
+    required property var savedAddressesModel
+    required property var recentRecipientsModel
+    /** Input property holds currently selected Fiat currency **/
+    required property string currentCurrency
+    /** Input function to format currency amount to locale string **/
+    required property var fnFormatCurrencyAmount
+
+    /** input property to decide if send modal is interactive or prefilled **/
+    property bool interactive: true
+
+    /** input property to set estimated time **/
+    property string estimatedTime
+    /** input property to set estimated fees in fiat **/
+    property string estimatedFiatFees
+    /** input property to set estimated fees in crypto **/
+    property string estimatedCryptoFees
+
+    /** property to set and expose currently selected account **/
+    property string selectedAccountAddress
```

Comment:
it's a bit more tricky to keep a single property for both input and output. There are two difficulties: preserving the bindings and avoiding the binding loops.

Maybe a nice solution here would be to keep all properties that are not readonly as input properties. Any further update can be requested by the `SimpleSendModal` using signals. Then the change can be validated and applied if needed.
WDYT?

Reply:
Same question for the other new components

Reply:
@alexjba before I started working on this me and @micieslak were brainstorming about it and how, having single proeprty for input and output, makes the api simpler and its something we see all over QT's components too. And Hence, decided to go this way, either can work, offcourse, if the base components were also implemented with single property for input & output it would have worked better here in the Modal and avoided some of the work around, but it seems to be a nice api from the Send Modal I feel.

Reply:
I agree! It would be great to have an input/output property! But there's one small detail in the Qt components. They're not breaking the external bindings and there's no binding loop 😄

I'm all for it if we can find a good solution. I think it's worth documenting it to be used in other components as well. But I'm not sure how we can achieve this with QML alone. @micieslak @caybro Any ideas?

Here's a Qt example on how the input/output properties work:

https://github.com/user-attachments/assets/b65004ed-176b-4646-a315-478957388b23

To break it down we have all these requirements ticked by the Qt components:
- Property value is updated with the internal value
- External bindings are preserved
- `onChanged` event is signaled